client/.env

```diff
@@ -0,0 +1,6 @@
+REACT_APP_API_KEY = AIzaSyCrucsAZVktDATv4zg3QT_FJ20zmg3T39U,
+REACT_APP_AUTH_DOMAIN = slack-standup-1556221821574.firebaseapp.com,
+REACT_APP_DATABASE_URL = https://slack-standup-1556221821574.firebaseio.com,
+REACT_APP_PROJECT_ID = slack-standup-1556221821574,
+REACT_APP_STORAGE_BUCKET = slack-standup-1556221821574.appspot.com,
+REACT_APP_MESSAGING_SENDER_ID = 357315122531,
```

Comment:
Should this be in a git ignore?

Reply:
Yes I will add to .gitignore and commit changes